rust/Cargo.toml

```diff
@@ -37,13 +37,13 @@ memmap = "0.7"
 rust-gpu-tools = { version = "0.5", optional = true, default-features = false }
 storage-proofs-porep = { version = "~12.0", default-features = false }
 fr32 = { version = "~5.0", default-features = false }
-fvm = { version = "1.2.0", default-features = false }
-fvm_shared = "0.9.0"
-fvm_ipld_blockstore = "0.1.1"
-fvm_ipld_encoding = "0.2.2"
+fvm = { git = "https://github.com/filecoin-project/ref-fvm", branch = "asr/nv17-release/v2" }
+fvm_shared = { git = "https://github.com/filecoin-project/ref-fvm", branch = "asr/nv17-release/v2" }
+fvm_ipld_blockstore = { git = "https://github.com/filecoin-project/ref-fvm", branch = "asr/nv17-release/v2" }
+fvm_ipld_encoding = { git = "https://github.com/filecoin-project/ref-fvm", branch = "asr/nv17-release/v2" }
```

Comment:
i think you want to point this at `release/v2` now

Reply:
Yeah, it'll need to be updated. I'm not sure whether we should point to `release/v2`, or tag a pre-release of the FVM and use that -- I plan on asking the FVM folks in sync today.

Reply:
We should point to an FVM pre-release, ideally. I can cut one (will be some 2.0.0-alpha.something).

Reply:
Sweet, please do, thank you!

Reply:
Done.